README.md

```diff
@@ -18,7 +18,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.google.cloud.tools:endpoints-framework-gradle-plugin:1.0.0-beta'
+    classpath 'com.google.cloud.tools:endpoints-framework-gradle-plugin:1.0.0-beta2'
```

Comment:
Is this going GA soon? If not, we could end up with a very large beta number, not sure if that's bad or not really.